contracts/libraries/TokenLib.sol

```diff
@@ -1,13 +1,13 @@
 pragma solidity ^0.5.0;
 
+import "./KindMath.sol";
```

Comment:
This may lead to adding back `KindMath` import after the merge.

Reply:
I think this will be solved by the other PR.

Reply:
That pr removes it but this pr ads it back. (it's in green not white). This pr is specifically adding it back.